services/api/src/typeDefs.js

```diff
@@ -329,6 +329,8 @@ const typeDefs = gql`
   }
 
   input AddFactsInput {
+    project: String
+    environment: String
```

Comment:
Would be good to have the ability to allow multiple input types for `addFacts`, but AFAIK unions only work for Object types currently?

Reply:
As discussed, I think introducing multiple mutations is probably right